Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -124,34 +124,45 @@
                                     FontFamily="{StaticResource FluentUIGlyphs}"
                                     FontSize="28"
                                     Glyph="{x:Bind Glyph}" />
-                                <StackPanel
+                                <Grid
                                     Grid.Column="1"
                                     Margin="12,0,0,0"
-                                    VerticalAlignment="Center"
-                                    Orientation="Vertical"
-                                    Spacing="8">
+                                    RowSpacing="8"
+                                    VerticalAlignment="Center">
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition/>
+                                        <RowDefinition Height="Auto"/>
+                                        <RowDefinition Height="Auto"/>
```

Comment:
```suggestion
                                        <RowDefinition Height="*"/>
                                        <RowDefinition Height="*"/>
                                        <RowDefinition Height="*"/>
```

Reply:
It does not change anything with `RowSpacing`, I guess only `Margin` works in this case

Reply:
@d2dyno1 Is there a reason you changed the StackPanel to a Grid?

Reply:
Hmmm, for some reason I though it wouldn't work with `StackPanel` lol. It works, but again if we want to have the name centered we have to ditch `Spacing` and replace it with `Margin` for every item